.github/configs/release-drafter.yml

```diff
@@ -0,0 +1,41 @@
+
+categories:
+  - title: '🚀 Features and Improvements'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🌈 Documentation Updates'
+    labels:
+      - 'documentation'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'bugfix'
+  - title: '🧰 Breaking Changes'
+    label: 'breaking'
+version-resolver:
+  major:
```

Comment:
Are we going to start incrementing the major version number now?

Reply:
I had this question too. One the one hand, we're using this in production in-house. On the other, would I consider it at a stable release if someone else other than us wanted to use it? I wanted to talk with you all about this to get thoughts on whether we should start using major versions for it.

Reply:
I guess I don't really have a strong opinion either way